ui/codeView.js

```diff
@@ -1730,11 +1730,19 @@ function getWindows(workspace) {
     }).filter((w, i, a) => !w._hackIsInactiveWindow && !w.skip_taskbar && a.indexOf(w) === i);
 }
 
+function is_speedwagon_window(metaWindow) {
```

Comment:
I guess we can drop this SpeedWagon thing entirely. We're dropping SpeedWagon from the desktop.

Reply:
I've removed all the speedwagon related code.